ceph-releases/infernalis/ubuntu/16.04/base/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update &&  apt-get install -y wget unzip uuid-runtime
 
 # Install Ceph
 RUN wget -q -O- 'https://download.ceph.com/keys/release.asc' | apt-key add -
-RUN echo deb http://ceph.com/debian-${CEPH_VERSION}/ trusty main | tee /etc/apt/sources.list.d/ceph-${CEPH_VERSION}.list
+RUN echo deb http://download.ceph.com/debian-${CEPH_VERSION}/ xenial main | tee /etc/apt/sources.list.d/ceph-${CEPH_VERSION}.list
```

Comment:
Since the previous says `trusty` (14.04), but the directory label is 16.04, I'm guessing this should be `wily`.  Otherwise, directory should be 16.10.  However, 16.10 isn't released yet, and 16.04 is the LTS release.